cli/installer/docker_compose.go

```diff
@@ -166,7 +166,8 @@ func getDockerComposeFileContents(ui UI, config configuration) []byte {
 		ui.Exit(fmt.Sprintf("cannot configure tracetest container: %s", err.Error()))
 	}
 
-	if config.Bool("tracetest.collector.install") {
+	isJaegerBackend := config.String("tracetest.backend.type") == "jaeger"
+	if config.Bool("tracetest.collector.install") && isJaegerBackend {
```

Comment:
still, this is not good. You need to fix the user input. Or even better, fix it so the collector works with any backend.
You can fix the fixOtelCollectorContainer. It will receive a project that is a parsed version of the docker-compose file. That file is fetch from the example for the selected backend. You can add a switch in that function so it correcty fixes the correct service.

Reply:
tbh, I don't think we need the `fixOtelCollectorContainer` function anymore. The example doesn't rely on the JAEGER_ENDPOINT env variable.

The old docker-compose file needed it.

Reply:
makes sense. If you test it and it works with at least jaeger and open search, we should be good to go

Reply:
Actually we don't need the jaeger env variable. We still need the volume configuration. Just added a new commit for that.